99-other-topics/009-custom-domain-cert.md

```diff
@@ -0,0 +1,97 @@
+# Using an externally-managed hostname
+
+## Background
+Every application running on Cloud Platform is able to use a hostname for their
+HTTP endpoints, under a pre-defined DNS zone. For exaxmple, on the `live-0`
+cluster, this would be `*.apps.cloud-platform-live-0.k8s.integration.dsd.io`. As
+long as it is defined on the `Ingress` resource, it works automatically with a
+wildcard TLS certificate.
+
+However, most applications will typically need to be served on a `gov.uk`
+hostname. These hostnames (or usually, DNS zones) are managed externally,
+relative to the cluster, and there is a number of actions in order to set them
+up for usage.
+
+## Setup
+
+### You do not have a DNS zone for the desired hostname
+[raise an issue with cloud-platform?]
```

Comment:
Absolutely, I left it as a placeholder so that we can decide exactly what we want to say/ask for but I suppose "providing as much information as possible" is acceptable?